main/templates/main/api_docs.html

```diff
@@ -160,7 +160,7 @@ <h2>DELETE /api/posts/&lt;post-slug&gt;</h2>
 </code></pre>
 
 
-    <h2>GET /api/posts/</h2>
+    <h2>GET /api/posts?s=&lt;term&gt;</h2>
```

Comment:
I was thinking if we could also make it explicit that the search term is optional. Do you think it's obvious?